<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PetoiSerialApp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/app/pic/logo.icns</normaloff>:/app/pic/logo.icns</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>981</width>
      <height>670</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSerial">
     <attribute name="title">
      <string>串口连结</string>
     </attribute>
     <widget class="QGraphicsView" name="dogView">
      <property name="geometry">
       <rect>
        <x>589</x>
        <y>10</y>
        <width>381</width>
        <height>620</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxSerialPort">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>561</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>串口连结</string>
      </property>
      <widget class="QSplitter" name="serialSetupSplitter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>541</width>
         <height>221</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_1">
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <widget class="QLabel" name="labelSerialPort">
           <property name="text">
            <string>串口号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxPortNumber">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelBaudRate">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxBaudRate">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelStopBits">
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxStopBits">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelDataBits">
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxDataBits">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_5">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labelParity">
           <property name="text">
            <string>奇偶校验</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxParity">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPushButton" name="buttonConnect">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>250</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRefresh">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string> 刷新</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxServosAdjust">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>561</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>舵机校准</string>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>207</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="buttonCleanPos">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>角度归零</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSavePos">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>541</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="buttonCalibration">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>开始校准</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxSteering">
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonInAngle">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonDeAngle">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>20</width>
        <height>620</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupTerminal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>410</y>
        <width>561</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>控制台</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>541</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="serialInputSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSendCmd">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>发送命令</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserTerminal"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPanel">
     <attribute name="title">
      <string>动作指令</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxActions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>动作姿态</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>611</width>
         <height>160</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="buttonPosStretch">
            <property name="text">
             <string>伸懒腰</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosButtomUp">
            <property name="text">
             <string>撅屁股</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosCheckRd">
            <property name="text">
             <string>左右看</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="buttonPosHi">
            <property name="text">
             <string>打招呼</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosPee">
            <property name="text">
             <string>撒尿</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosPushUp">
            <property name="text">
             <string>俯卧撑</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="buttonPosSit">
            <property name="text">
             <string>坐下</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosSleep">
            <property name="text">
             <string>睡觉</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosRest">
            <property name="text">
             <string>休息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="buttonPosStepping">
            <property name="text">
             <string>原地踏步</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosBackFlip">
            <property name="text">
             <string>后空翻</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosBunnyJump">
            <property name="text">
             <string>兔子跳</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxServosPanel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>设备控制</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="buttonCtrlRestPos">
          <property name="text">
           <string>恢复站立</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCtrlBalance">
          <property name="text">
           <string>平衡姿态</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>291</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="buttonCtrlGyro">
          <property name="text">
           <string>关闭陀螺仪</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCtrlServos">
          <property name="text">
           <string>关闭舵机</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="textBrowserSpecification">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Up: w	Shift: run&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Down: s	Ctrl: crawl&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Left: a&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Right: d&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
